### ISSUE TITLE
"Your client is using unsupported sync protocol (10, supported version: 9)

### ISSUE THREAD

**Issue by sevi1603** [kuklinistvan/docker-anki-sync-server#14]
Syncing with AnkiDroid works

When trying to use the Windows Desktop client (Version 2.1.22) and the SyncRedirector plugin with this config:

{
    "msyncUrl": "http://192.168.2.80:27701/msync/",
    "syncUrl": "http://192.168.2.80:27701/sync/"
}

(Username and password has been put into the authentication pop up.)

I'm getting the following error message whenever a sync is triggered:
"Your client is using unsupported sync protocol (10, supported version: 9)

Followed by this error:
AnkiWeb ID or password was incorrect; please try again.

Logs from docker container:
[2020-04-03 10:23:02,669]:INFO:ankisyncd.CollectionThread[user]:Running meta(*[], **{'v': 10, 'cv': 'ankidesktop,2.1.22 (0ecc189a),win:10'})
[2020-04-03 10:24:32,762]:INFO:ankisyncd.ThreadingCollectionManager:Monitor is closing collection on inactive CollectionThread[user]

**Comment by sevi1603** [comment]
I noticed in the changelog of the Windows Anki Client 2.1.22 some sync related changes. When using the 2.1.15-standard client, syncing works.
Seems like the server hasn't been updated yet

**Comment by kuklinistvan** [comment]
Hello,

I'm sorry for the late response.

I've noticed too that Anki tends to change its synchronization protocol quite often between patch version number updates. With the help of [tsudoko](https://github.com/tsudoko) I'm supporting (well, doing my best to support) the actual version stated in the [README.md](https://github.com/kuklinistvan/docker-anki-sync-server/tree/13232886c8105295337815c8844befd6b52e0136#tested-and-works-on). At the moment these are AnkiDesktop version 2.1.19 and AnkiDroid version 2.9.1.

Yesterday I've put download links to the versions I've tested the synchronization with. I've also created a mirror for these executables in case those links become broken.

Happy syncing :)
